packages/react/src/CartCost.tsx

```diff
@@ -0,0 +1,42 @@
+import {Money} from './Money.js';
+import {useCart} from './CartProvider.js';
+
+export interface CartCostProps {
+  /** A string type that defines the type of cost needed. Valid values: `total`, `subtotal`, `tax`, or `duty`. */
+  amountType?: 'total' | 'subtotal' | 'tax' | 'duty';
+  /** A function that takes an object return by the `useMoney` hook and returns a `ReactNode`. */
```

Comment:
```suggestion
   /** Any `ReactNode` elements. */
```

Reply:
uh. This was the original doc 🤔 haha thanks I'll update